### ISSUE TITLE
FormData should not be automatically instrumented

### ISSUE THREAD

**Issue by backjo** [justindsmith/opentelemetry-instrumentations-js#43]
Currently, this library automatically tracks formData values as attributes on spans. This should at the least be configurable - sensitive data such as passwords etc may be passed by applications through formData and that data should not be recorded

**Comment by justindsmith** [comment]
There is a configurable object that should specify which formData values get added as attributes (the `actionFormDataAttributes` configuration specified in the README). By default that object is `{ "_action": true }` so that ONLY the `_action` formData value gets added by default.

Here's the code that is parsing the formData: https://github.com/justindsmith/opentelemetry-instrumentations-js/blob/main/packages/instrumentation-remix/src/instrumentation.ts#L425-L434

Are you seeing a different behavior with the default configuration? What version of remix are you using?

**Comment by backjo** [comment]
hey @justindsmith - we are! We're using the latest version along with Remix 1.19.2 - we're seeing something being emitted in the span along the lines of:
```json
"formData": {
    "undefined": "sensitive-value"
}
```

Looking at the code you referenced:
```ts
const formData = await clonedRequest.formData();
const { actionFormDataAttributes: actionFormAttributes } = plugin.getConfig();
formData.forEach((value, key) => {
  if (actionFormAttributes[key] !== false) {
    const keyName = actionFormAttributes[key] === true ? key : actionFormAttributes[key];
    span.setAttribute(`formData.${keyName}`, value.toString());
  }
});
```
If `actionFormAttributes[key]` is undefined, it will pass the strict equality check against `false` and then set the attribute as `formData.undefined` to the value.

**Comment by backjo** [comment]
hey @justindsmith  - opened up a PR with a fix and a test case to help reproduce the error (if removing the fix)

**Comment by justindsmith** [comment]
Very good catch! Thanks for the PR, it's running tests now. If all succeeds I can push a release in a few hours. Thank you for fixing and adding another test case for this!

**Comment by justindsmith** [comment]
Published as [v0.5.1](https://www.npmjs.com/package/opentelemetry-instrumentation-remix/v/0.5.1)

Thank you for the report and fix! :)
